core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -38,7 +38,12 @@ package object config {
     ConfigBuilder("spark.driver.userClassPathFirst").booleanConf.createWithDefault(false)
 
   private[spark] val DRIVER_MEMORY = ConfigBuilder("spark.driver.memory")
+    .doc("Amount of memory to use for the driver process, i.e. " +
```

Comment:
Can you also check for other configs in this file and add `checkValue`s as much as possible? Thanks!